tests/feature/test_tags.py

```diff
@@ -226,3 +226,47 @@ def _():
 
     result = pytester.runpytest("-m", "tag2", "-vv")
     result.assert_outcomes(passed=1, deselected=1)
+
+
+def test_tags_against_multiple_examples_tables(pytester):
```

Comment:
to me this can be removed, and instead bring these assertions to the test you made in test_multiple_outlined.

Reason is that this test is really about the scenario outline functionality. I'm not even sure that this `test_tag.py` has a reason to exist. It's test items should just go in the test module where we assert the behaviour of Feature and Scenarios (and then Examples here)